tests/setup_cfg_fmt_test.py

```diff
@@ -674,12 +774,15 @@ def test_guess_python_requires_tox_ini_py310(tmpdir):
         'classifiers =\n'
         '    Programming Language :: Python :: 3\n'
         '    Programming Language :: Python :: 3 :: Only\n'
+        '    Programming Language :: Python :: 3.7\n'
+        '    Programming Language :: Python :: 3.8\n'
+        '    Programming Language :: Python :: 3.9\n'
         '    Programming Language :: Python :: 3.10\n'
         '    Programming Language :: Python :: 3.11\n'
         '    Programming Language :: Python :: Implementation :: CPython\n'
         '\n'
         '[options]\n'
-        'python_requires = >=3.10\n'
+        'python_requires = >=3.7\n'
     )
```

Comment:
I think this one is not quite right -- I think the defaulting is happening too early so it's ignoring all the tox / etc. settings?

Reply:
Ah you're right the logic I have is "specify nothing: minimum is **min**(3.7, python_requires minimum, tox minimum, ...)" not "[...] **max**(3.7, [...]"